Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM tiangolo/uwsgi-nginx-flask:python3.6 as step1
+COPY . /app
+RUN pip install --upgrade pip
+RUN pip install -r ./requirements.txt
+ENV ENVIRONMENT production
+CMD ["python","/data_process/tsv_to_dataframe.py"]
+
+FROM tiangolo/uwsgi-nginx-flask:python3.6 as step2
```

Comment:
why not do FROM step1 image, which already has all pip dependencies in place? this would reduce build time and shorten the Dockerfile

Reply:
Hey that's not working as we are required to run multiple scripts in an order. Will try it if possible

Reply:
Okay I got a work around to this will update the docker file